addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -13,6 +13,15 @@
     flex: 1;
     flex-direction: row;
     flex-wrap: wrap;
+
+    &.o-has-active-button {
+        border-bottom: $border-width solid;
+    }
+}
+
+.o_ChatterTopbar_button {
+    border-bottom: none;
```

Comment:
Is this needed? I don't think there is any border in the first place.

Reply:
Yes it is, it's forcing the border bottom to not appear when button is active, the o-active modifier was not really needed because it should never have any border bottom

Reply:
I tested without this rule and I don't see any problem, I still don't understand why have it as nobody else is setting border bottom on this element, it doesn't need to be overridden.

Reply:
It would be best if you show me with screen share then.

Reply:
Well, I don't know why but now the border does not appear anymore so this rule seems indeed useless